### ISSUE TITLE
Rename Python package because cedarpolicy is too similar to an existing project

### ISSUE THREAD

**Issue by skuenzli** [k9securityio/cedar-py#11]
```
💥 maturin failed
  Caused by: 💥 Failed to upload "cedarpolicy-0.1.0-cp310-cp310-macosx_10_7_x86_64.whl" (1.9 MB)
  Caused by: Failed to upload the wheel with status 400: <html>
 <head>
  <title>400 The name 'cedarpolicy' is too similar to an existing project. See https://pypi.org/help/#project-name for more information.</title>
 </head>
 <body>
  <h1>[40](https://github.com/k9securityio/cedar-py/actions/runs/5480638520/jobs/9984137940#step:3:43)0 The name 'cedarpolicy' is too similar to an existing project. See https://pypi.org/help/#project-name for more information.</h1>
  The server could not comply with the request since it is either malformed or otherwise incorrect.<br/><br/>
The name &#x27;cedarpolicy&#x27; is too similar to an existing project. See https://pypi.org/help/#project-name for more information.
```

Looks like someone created `cedar-policy` on July 3rd (three days ago):
https://pypi.org/project/cedar-policy/

https://pypi.org/project/cedar-policy/0.0.1/#history

**Comment by skuenzli** [comment]
Renamed to `cedarpy`

**Comment by skuenzli** [comment]
Published `cedarpy` 0.1.0!

https://pypi.org/project/cedarpy/
